Change `str_to_snake_case` to handle capitalized words (#223)
mirumee/ariadne-codegen: CHANGELOG.md

- Fixed generating results for nullable fields with nullable directives.
- Changed `include_comments` option to accept enum value, changed default to `"stable"`, deprecated boolean support. Added `get_file_comment` plugin hook.

> - Changed `str_to_snake_case` utility to correctly handle capitalized words. This is a breaking change, digits are now preceded by an underscore.

Reviewer: ```suggestion
- Changed `str_to_snake_case` utility to correctly handle capitalized words.
- Digits in Python names are now preceded by an underscore (breaking change).
```
